### ISSUE TITLE
Add opt-in tracking

### ISSUE THREAD

**Issue by MaximilianGaedig** [16arpi/Fox-RSS#9]
Izzydroid currently displays a warning because the app is sending logs without explicit permission or even notice, would be nice if you could add a prompt asking the user if they want to send the information to you / a notice

**Comment by Mattrazol** [comment]
Why not remove the logs completely? 😄

**Comment by MaximilianGaedig** [comment]
> Why not remove the logs completely? 😄

It's in it's early dev phases, it's understandable that the developer wants feedback.

**Comment by 16arpi** [comment]
I'll find a solution later, meanwhile please let me drain your private data 😈😈😈

**Comment by Mattrazol** [comment]
😂😂😂

**Comment by Mattrazol** [comment]
By the way congratulations for the app! It is very well done! And it's the best looking RSS reader IMHO 😄

**Comment by MaximilianGaedig** [comment]
> By the way congratulations for the app! It is very well done! And it's the best looking RSS reader IMHO 😄

I agree, really nice concept, I am jealous of such a cool idea with the tinder-like swiping

**Comment by 16arpi** [comment]
Thx :) but you should know that the idea isn't totally from me. Le Monde newspaper has an app which use this tinder system to present their articles and I really enjoy it but I'm not a big fan of Le Monde so I made the same app for all my rss feeds 😁
